Dockerfile

```diff
@@ -45,15 +45,16 @@ RUN sudo chown -R pi:pi ~/catkin_ws
 # Install ROS boostrapping tool
 RUN sudo apt-get update && sudo apt-get install --no-install-recommends -y -q \
     python-pip python-rosdep
+
```

Comment:
I should probably file a separate issue, but the Dockerfile in `develop` is currently broken, mainly because it got out of sync with `scripts/install_dev`. I've got a working version [here (summary in the commit message)](https://github.com/davoclavo/openag_brain/commit/5f7e97a7d651c388b99227a591d39f64a3d2e1c1) but was trying to see if I could refine it more before submitting a PR, let me know if I should submit one soon before this PR gets merged or if we can push it back for a future release.
My changes make the docker build rely on the `scripts/install_dev` which sets up almost everything; it is not the best way to solve this, but it works in the interim. The reason why I say is not the best solution is that then docker can't leverage caching between commands, so every time a change is made in the `install_dev` script, the whole step has to be re-built, which takes a long time.